common/src/main/java/com/wynntils/features/user/map/MapFeature.java

```diff
@@ -81,4 +104,66 @@ public class MapFeature extends UserFeature {
     public List<Class<? extends Model>> getModelDependencies() {
         return List.of(MapModel.class);
     }
+
+    @SubscribeEvent
+    public void onRightClick(PlayerInteractEvent.RightClickBlock event) {
+        if (!autoWaypointChests) return;
+
+        lastChestPos = event.getPos();
```

Comment:
check if chest?

Reply:
no need to imo

Reply:
Could be caused by lag to have lastChestPos not match screen

Reply:
Maybe, but this is the best we can do. Legacy does the same. Ill add block checking later

Reply:
Added the check.